app/types/datetime.go

```diff
@@ -5,8 +5,6 @@ import (
 	"time"
 )
 
-// Doc is mainly in the "value" file :-)
```

Comment:
It is still in the other types (bool and string)

Reply:
fixed